src/main/java/org/truffleruby/core/exception/ExceptionNodes.java

```diff
@@ -153,9 +152,9 @@ public DynamicObject exceptionErrnoError(DynamicObject message, int errno, Dynam
             final String errorMessage;
             if (message != nil()) {
                 if (RubyGuards.isRubyString(location)) {
-                    errorMessage = " @ " + location.toString() + " - " + RopeOperations.decodeRope(StringOperations.rope(message));
+                    errorMessage = " @ " + location + " - " + StringOperations.getString(message);
```

Comment:
Can you make this `StringOperations.getString(location)` as well, please? I'm trying to limit the cases where `toString()` on a Ruby String or Rope is called.

Reply:
Ah, I confused location and message.
Anyway, this stuff should be in Ruby.

Reply:
Done and moved to Ruby.